docs: fixed typos (#2680)
VictoriaMetrics/VictoriaMetrics: README.md

* [digitalocean_sd_config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#digitalocean_sd_config)
* [http_sd_config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#http_sd_config)

> If you need to support for other `*_sd_config` types file free to open a [feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues).

Reviewer: ```suggestion
If you need to support for other `*_sd_config` types feel free to open a [feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues).
```

> If you need to support for other `*_sd_config` types feel free to open a [feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues).

Reviewer 2: ```suggestion
File a [feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues) if you need support for other `*_sd_config` types.
```

Optional `start` and `end` args may be added to the request in order to limit the time frame for the exported data. These args may contain either
unix timestamp in seconds or [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) values.
For example:
```bash

> 1. http://<victoriametrics-addr>:8428/api/v1/export?match[]=<timeseries_selector_for_export>&start=1654543486&end=1654543486

Reviewer 2: ```suggestion
curl http://<victoriametrics-addr>:8428/api/v1/export -d 'match[]=<timeseries_selector_for_export>' -d 'start=1654543486' -d 'end=1654543486'
```

Optional `start` and `end` args may be added to the request in order to limit the time frame for the exported data. These args may contain either
unix timestamp in seconds or [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) values.
For example:
```bash

> 1. http://<victoriametrics-addr>:8428/api/v1/export?match[]=<timeseries_selector_for_export>&start=1654543486&end=1654543486

Reviewer 2: ```suggestion
curl http://<victoriametrics-addr>:8428/api/v1/export/csv -d 'format=<format>' -d 'match[]=<timeseries_selector_for_export>' -d 'start=1654543486' -d 'end=1654543486'
```

Optional `start` and `end` args may be added to the request in order to limit the time frame for the exported data. These args may contain either
unix timestamp in seconds or [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) values.
For example:
```bash
1. http://<victoriametrics-addr>:8428/api/v1/export?match[]=<timeseries_selector_for_export>&start=1654543486&end=1654543486

> 2. http://<victoriametrics-addr>:8428/api/v1/export?match[]=<timeseries_selector_for_export>&start=2022-06-06T19:25:48+00:00&end=2022-06-06T19:29:07+00:00

Reviewer 2: ```suggestion
curl http://<victoriametrics-addr>:8428/api/v1/export/native -d 'match[]=<timeseries_selector_for_export>' -d 'start=2022-06-06T19:25:48+00:00' -d 'end=2022-06-06T19:29:07+00:00'
```

at `http://<victoriametrics-addr>:8428/federate?match[]=<timeseries_selector_for_federation>`.

Optional `start` and `end` args may be added to the request in order to scrape the last point for each selected time series on the `[start ... end]` interval.
`start` and `end` may contain either unix timestamp in seconds or [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) values.
For example:
```bash

> 1. http://<victoriametrics-addr>:8428/api/v1/export?match[]=<timeseries_selector_for_export>&start=1654543486&end=1654543486

Reviewer 2: ```suggestion
curl http://<victoriametrics-addr>:8428/federate -d 'match[]=<timeseries_selector_for_export>' -d 'start=1654543486' -d 'end=1654543486'
```

> 2. http://<victoriametrics-addr>:8428/api/v1/export?match[]=<timeseries_selector_for_export>&start=2022-06-06T19:25:48+00:00&end=2022-06-06T19:29:07+00:00

Reviewer 2: ```suggestion
curl http://<victoriametrics-addr>:8428/federate -d 'match[]=<timeseries_selector_for_export>' -d 'start=2022-06-06T19:25:48+00:00' -d 'end=2022-06-06T19:29:07+00:00'
```

```

> By default, the last point on the interval `[now - max_lookback ... now]` is scraped for each time series. The default value for `max_lookback` is `5m` (5 minutes), but it can be overridden.

Reviewer 2: ```suggestion
By default, the last point on the interval `[now - max_lookback ... now]` is scraped for each time series. The default value for `max_lookback` is `5m` (5 minutes), but it can be overridden with `max_lookback` query arg.
```
